airflow/configuration.py

```diff
@@ -421,6 +421,25 @@ def parameterized_config(template):
 conf = AirflowConfigParser()
 conf.read(AIRFLOW_CONFIG)
 
+if conf.getboolean('webserver', 'rbac'):
+    with open(os.path.join(_templates_dir, 'default_webserver_config.py')) as f:
+        DEFAULT_WEBSERVER_CONFIG = f.read()
+
+    WEBSERVER_CONFIG = AIRFLOW_HOME + '/webserver_config.py'
+
+    if not os.path.isfile(WEBSERVER_CONFIG):
+        log.info('Creating new FAB webserver config file in: %s', WEBSERVER_CONFIG)
+        with open(WEBSERVER_CONFIG, 'w') as f:
+            f.write(DEFAULT_WEBSERVER_CONFIG)
+
+    if 'WEBSERVER_CONFIG' not in os.environ:
+        if os.path.isfile(expand_env_var('~/webserver_config.py')):
+            WEBSERVER_CONFIG = expand_env_var('~/webserver_config.py')
+        else:
+            WEBSERVER_CONFIG = AIRFLOW_HOME + '/webserver_config.py'
+    else:
+        WEBSERVER_CONFIG = expand_env_var(os.environ['WEBSERVER_CONFIG'])
```

Comment:
Could this entire block not be removed if this was a normal setting in airflow.cfg, and then the default env of `AIRFLOW__WEBSERVER__CONFIG` could be used.